autogpt_platform/backend/backend/blocks/basic.py

```diff
@@ -35,23 +33,21 @@ class Output(BlockSchema):
     def __init__(self):
         super().__init__(
             id="1ff065e9-88e8-4358-9d82-8dc91f622ba9",
-            description="This block forwards an input value as output, allowing reuse without change.",
+            description="This is a special block that provides a value when triggered. The value can be reused multiple times.",
             categories={BlockCategory.BASIC},
             input_schema=StoreValueBlock.Input,
             output_schema=StoreValueBlock.Output,
             test_input=[
-                {"input": "Hello, World!"},
-                {"input": "Hello, World!", "data": "Existing Data"},
+                {"input": "Hello, World!", "value": "Existing Data"},
```

Comment:
`input` changed to `trigger` in `class Input` but not here and CI fails because of that.
```suggestion
                {"trigger": "Hello, World!", "value": "Existing Data"},
```

Reply:
Thanks